apps/glusterfs/volume_entry.go

```diff
@@ -420,11 +420,35 @@ func (v *VolumeEntry) cleanupCreateVolume(db wdb.DB,
 
 	// from a quick read its "safe" to unconditionally try to delete
 	// bricks. TODO: find out if that is true with functional tests
-	DestroyBricks(db, executor, brick_entries)
+	reclaimed, _ := DestroyBricks(db, executor, brick_entries)
```

Comment:
Because we're no longer discarding the entire result of DestroyBricks, lets at least capture and log any errors from the function while were here.

Reply:
added logging if DestroyBricks fails